src/components/ItemFilters/ItemFilters.tsx

```diff
@@ -160,7 +131,55 @@ const FiltersContainer = ({
             onSubmit={handleYearSubmit}
           />
         </Box>
-      </Box>
+      </>
+    )
+  }
+
+  // function passed to filterElements map callback for generating renderMultiSelect prop of MultiSelectGroup
+  const renderMultiSelect = (
+    checkboxGroup: FilterCheckboxGroup,
+    appliedFilterData = {}
+  ) => {
+    return (
+      <MultiSelect
+        buttonText={checkboxGroup.name}
+        id={`${checkboxGroup.id}-multi-select`}
+        data-testid={`${checkboxGroup.id}-multi-select`}
+        items={checkboxGroup.items}
+        key={checkboxGroup.id}
+        width="fitContent"
+        __css={{ flex: 1 }}
+        onChange={(e: React.ChangeEvent<HTMLInputElement>): void => {
+          console.log(e.target.value)
+        }}
+        onClear={() => {
+          console.log(checkboxGroup.id)
+        }}
+        selectedItems={appliedFilterData}
+      />
+    )
+  }
+
+  return (
+    <>
+      <FilterBarInline
+        id="item-filters-container"
+        data-testid="item-filters-container"
+        p="s"
```

Comment:
Following the docs, it's the parent that should have the background color and the padding (it should be `p="inset.wide"`). https://nypl.github.io/nypl-design-system/reservoir/v3/?path=/docs/components-form-elements-filterbarinline--docs
Check the UI Containers section.